_posts/2022-08-12-WX_Blog2.md

```diff
@@ -0,0 +1,73 @@
+# Hello, R2-D2
+
+"May Day! May Day! Darth Vader strikes! His death star will shoot death rays at us in no time!"
+"Oh, R2-D2 has the key to millenium falcon! Let's ask him to open the door of the star ship!"
+"Hey R2-D2! Hurry up! The lineage of royal family lies in your hand!"
+
+`import R2-D2`
+
+"Open the door please!"
+
+`name error! 'Open the door' is not defined`
+
+"Oh my! what should we do now? what does it mean not defined?"
+"Maybe it's written in Python? Let's try it again!"
+
+`import R2-D2.open_the_door`
+
+"No error? Now what?"
+"OK I see. R2-D2 is the package. open_the_door.py is the module. We still need to call the function!"
+
+`R2-D2.open_the_door.open_the_door_for_her_majesty_and_escape_with_star_ship`
+
+"It's working! RD-D2 is opening the door!
+
+Boom~~~!!! Too late...
+
+# What about the introduction?
+
+For a book there is a preface. For an article there is introduction. For a software package there is `__init__.py`.
+As total amateurs, we would ideally want the module to perform the following functions:
+1. Give a succint but reasonable introduction to the package. "Imported Monte Carlo!" is what we have now. It's pretty clear, but could be even more illustrative. We might as well go with a sentence that explains the purpose of the function, like "Imported Monte Carlo for Lennard Jones (LJ) Simulation!".
+2. Import all the necessary modules and provide a little hint, if possible. Ultimately we need to be user friendly when developing a software. It's important to not let the users guess what's inside the package like peeking into a black box. But also, if we load too many modules the users might have another level of bad experience -- slow motion.
```

Comment:
Ultimately, ... missing a comma. I'm not quite sure what do you mean by slow-motion.